Makefile

```diff
@@ -155,11 +155,12 @@ bench: node_modules $(GEN)/protobuf-bench $(BUILD)/protobuf ## Benchmark code si
 perf: $(BUILD)/protobuf-test
 	npm run -w packages/protobuf-test perf
 
-.PHONY: bootstrapwkt
-bootstrapwkt: $(BUILD)/upstream-protobuf $(BUILD)/protoc-gen-es node_modules ## Generate the well-known types in @bufbuild/protobuf
-	npm run -w packages/protobuf bootstrap
+.PHONY: bootstrap
+bootstrap: $(BUILD)/upstream-protobuf $(BUILD)/protoc-gen-es node_modules ## Bootstrap well-known types and edition features-set defaults in @bufbuild/protobuf from upstream protobuf
+	npm run -w packages/protobuf bootstrap:wkt
+	npm run -w packages/protobuf bootstrap:featureset-defaults
 	@# If the generated code differs, use it, and run tests again
-	npm run -w packages/protobuf bootstrap-diff || $(MAKE) build test format lint
+	#npm run -w packages/protobuf bootstrap-diff || $(MAKE) build test format lint
```

Comment:
Did you mean to have this commented out?

Reply:
Yes, this is no longer used. The bootstrapping process is getting too complex and isn't portable to turborepo. With this PR, updating to a new upstream release means you have to run `make bootstrap` and then `make` again to run tests with the updated code. Documented [here](https://github.com/bufbuild/protobuf-es/tree/main/packages/upstream-protobuf). Diffcheck in CI will catch if you forgot.